conformance/tests/basic/inferencepool_lifecycle.yaml

```diff
@@ -0,0 +1,95 @@
+# Basic InferencePool lifecycle test.
+# This manifest defines the minimal required fields to create valid
+# InferencePool and HTTPRoute resources, which the lifecycle test 
+# will use to verify that the controller recognizes and accepts the resource.
+
+# --- Minimal Backend Deployment (using agnhost echo server) ---
+# This Deployment provides Pods for the InferencePool to select.
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: infra-backend-v1-deployment
+  namespace: gateway-conformance-app-backend
+  labels:
+    app: infra-backend-v1
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: infra-backend-v1
+  template:
+    metadata:
+      labels:
+        app: infra-backend-v1
+    spec:
+      containers:
+      - name: agnhost-echo
+        image: k8s.gcr.io/e2e-test-images/agnhost:2.39
+        args:
+        - serve-hostname
+        - --http-port=8080
+        ports:
+        - name: http
+          containerPort: 8080
+        readinessProbe:
+          httpGet:
+            path: /
+            port: 8080
+          initialDelaySeconds: 3
+          periodSeconds: 5
+          failureThreshold: 2
+
+---
+# --- InferencePool Definition for Lifecycle Test ---
+apiVersion: inference.networking.x-k8s.io/v1alpha2
+kind: InferencePool
+metadata:
+  # This name must match the 'poolNN' variable defined in the
+  # conformance/tests/basic/inferencepool_lifecycle.go test file.
+  name: inferencepool-lifecycle
+  # This namespace should be one created by the base manifests.
+  namespace: gateway-conformance-app-backend
+spec:
+  # --- Selector (Required) ---
+  # Selects the Pods belonging to this pool.
+  selector:
+    app: "infra-backend-v1"
+
+  # --- Target Port (Required) ---
+  # The port the model server container (agnhost in this case) listens on.
+  targetPortNumber: 8080 # Matches agnhost's http-port
+
+  # --- Extension Reference ---
+  # GKE-specific configuration reference.
+  extensionRef:
+    # group: "" # Optional
+    # kind: Service # Optional
+    name: infra-backend-v1-epp
+
+---
+# --- HTTPRoute Definition for Lifecycle Test ---
+# This HTTPRoute links the InferencePool to the shared Gateway,
+# allowing the InferencePool to receive an "Accepted" status condition.
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: httproute-for-inferencepool-lifecycle
```

Comment:
```suggestion
  name: inferencepool-lifecycle
```

Reply:
based on discussion in [795#issuecomment-2873371974](https://github.com/kubernetes-sigs/gateway-api-inference-extension/pull/795#issuecomment-2873371974) removing the lifecycle test.